_data/simple-icons.json

```diff
@@ -518,7 +518,7 @@
         {
             "title": "Angular",
             "hex": "DD0031",
-            "source": "https://angular.io/assets/images/logos/angular/angular_solidBlack.svg"
+            "source": "https://angular.io/presskit"
```

Comment:
Good find 💯 
We could also add the license for this one!

I even wonder if we should update the icon for their "One Color Logo" since they provide one... Might open a new issue to discuss this.

Reply:
This one is not entirely resolved @jorgeamadosoria. We can add the license entry for this one as mentioned in their presskit.

Reply:
That's done now.